packages/components/psammead-most-read/src/item.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
@EinsteinNjoroge, question, why is this file name `item.jsx` ? and the `main` parameter in `package.json` still `index.js`

Reply:
The idea is to have an `index.js` file which will export a list of most read items. That will be the next step in this component's roadmap.

Reply:
thanks, that makes sense